docs/pages/workflow/continuous-native-generation.mdx

```diff
@@ -4,90 +4,91 @@ description: Learn about Continuous Native Generation (CNG) and how it works.
 
-1. The [Expo Config][expo-config] file (`app.json`, `app.config.js`).
+1. The [app config](/workflow/configuration/) file.
 2. Arguments passed to the `npx expo prebuild` command.
-3. Version of `expo` that's installed in the project and its corresponding [prebuild template](#templates).
-4. [Autolinking][autolinking], for linking [native modules][native-modules] found in the `package.json`.
+3. Version of `expo` that's installed in the project and corresponding [prebuild template](https://github.com/expo/expo/tree/main/templates/expo-template-bare-minimum).
+4. [Autolinking](/more/glossary-of-terms#autolinking), for linking [native modules](/more/glossary-of-terms/#native-module) found in the **package.json**.
 5. Native Subscribers, for reducing native code side-effects in entry files.
-6. EAS Credentials for code signing additional targets and entitlments.
+6. EAS Credentials for code signing additional targets and entitlements.
 
-This boils down to a workflow where a user can express any native application with the `app.json` and generate that project--continuously--by running `npx expo prebuild`.
+This boils down to a workflow where a user can express any native application with the app config and generate that project continuously &mdash; by running `npx expo prebuild`.
 
-## Community Adoption
+## Community adoption
 
-Many complex libraries and services already support CNG via Expo Prebuild, [MapBox](https://github.com/rnmapbox/maps), [OneSignal](https://github.com/OneSignal/onesignal-expo-plugin), [Stripe](https://github.com/stripe/stripe-react-native), and [React Native Firebase](https://rnfirebase.io/#expo) to name a few examples.
+Many complex libraries and services already support CNG via Expo Prebuild such as, [MapBox](https://github.com/rnmapbox/maps), [OneSignal](https://github.com/OneSignal/onesignal-expo-plugin), [Stripe](https://github.com/stripe/stripe-react-native), and [React Native Firebase](https://rnfirebase.io/#expo).
 
 Libraries that don't have first-party integration are also supported in the [community Config Plugins repository](https://github.com/expo/config-plugins/).
 
-But the best part; Expo Config Plugins have first-class support for building locally and adding to your project without any third-party intervention or NPM publishing. This enables you to move fast and patch things!
+Expo Config Plugins have first-class support for building locally and adding to your project without any third-party intervention or npm publishing. This enables you to move fast and patch things!
 
 <a
   aria-label="Expo Config Plugins downloads"
   href="http://www.npmtrends.com/@expo/config-plugins"
   target="_blank">
-  <img
-    alt="Downloads"
+  <ImageSpotlight
+    alt="Expo Config Plugins downloads."
     src="https://img.shields.io/npm/dm/@expo/config-plugins.svg?style=flat-square&labelColor=gray&color=33CC12&label=Downloads"
+    style={{ maxWidth: 480 }}
```

Comment:
I still think that this asset is too big, and in general, I don't like the idea of using shield in page content.

<img width="1161" alt="Screenshot 2023-06-19 at 10 39 22" src="https://github.com/expo/expo/assets/719641/cd8b249e-27f6-4d98-890a-ebf5ed474345">

Reply:
You're right. Making the maxWidth to `200` makes it a bit more presentable:

<img width="894" alt="CleanShot 2023-06-19 at 14 13 01@2x" src="https://github.com/expo/expo/assets/10234615/637f06ec-ec5d-4022-8e27-6a07b74d9e82">


What do you suggest as a replacement? A text link?

Reply:
Doesn't have a good replacement idea, thus did not mark it as blocker for the PR. Shrinking should be sufficient for now, we can think about alternate solution in the future.

Reply:
Thank you!